lib/t/push_notifications/dispatch_job.ex

```diff
@@ -51,18 +51,9 @@ defmodule T.PushNotifications.DispatchJob do
   end
 
   defp handle_type(type, args) when type in ["timeslot_offer", "timeslot_accepted"] do
-    %{"match_id" => match_id, "receiver_id" => receiver_id} = args
-
-    if match = alive_match(match_id) do
-      %Matches.Match{user_id_1: uid1, user_id_2: uid2} = match
-
-      picker_id =
-        if(receiver_id == uid1) do
-          uid2
-        else
-          uid1
-        end
+    %{"match_id" => match_id, "receiver_id" => receiver_id, "picker_id" => picker_id} = args
```

Comment:
this function is designed for two types of pushes: `["timeslot_offer", "timeslot_accepted"]`

`"timeslot_offer"` push doesn't have `"picker_id"` in its args: 
![image](https://user-images.githubusercontent.com/16062322/142803904-b67426e1-19c5-41f8-897b-164746580b68.png)

therefore this line would crash. please be more attentive and double-check your work

Reply:
ok, will do, sorry
add picker